packages/frontend-v2/pages/_app.tsx

```diff
@@ -5,6 +5,7 @@ import { theme } from "../utils";
 import "react-toastify/dist/ReactToastify.min.css";
 import { ToastContainer } from "react-toastify";
 import { ErrorBoundary } from "../components";
+import "@fontsource/montserrat-alternates";
```

Comment:
for anyone else, looks like this is the preferred way to install fonts for chakra https://chakra-ui.com/docs/styled-system/recipes/using-fonts probably good to add this to docs @aryanshah701

Reply:
yeap, will do!